src/core/infrastructure/entities/Auth.ts

```diff
@@ -1,16 +1,20 @@
-import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { Column, Entity, Unique, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
 import { UserRecord } from "./User";
 
 @Entity({ name: "user_sessions" })
+@Unique(["id", "token"])
```

Comment:
I see what you're trying to do here. I think you're trying to make userId's and token's unique. 

The ID is always unique as it is. You will never have the same ID twice. As a result, this is redundant.


I think what you're trying to do will require you to create a relationship with the User Entity table. Each token will pertain to a userId. 

Each token should also be unique, such that users don't have the same tokens between users.